pom.xml

```diff
@@ -2344,6 +2339,7 @@
       <id>fast</id>
       <properties>
         <skipTests>true</skipTests>
+        <asciidoctor.skip>true</asciidoctor.skip>
```

Comment:
@joakime finally this is a real issue :) 
```
mvn -Pfast -ntp -V -B clean install -DskipTests -T6 -e
```
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-assembly-plugin:3.6.0:single (make-assembly) on project jetty-documentation: Failed to create assembly: Error creating assembly archive html: archive cannot be empty -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-assembly-plugin:3.6.0:single (make-assembly) on project jetty-documentation: Failed to create assembly: Error creating assembly archive html: archive cannot be empty
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:333)

````

@lorban this is the issue with perf build

Reply:
@olamy the `-Pfast` is skipping the steps needed for the documentation build.
Can `-Pfast` somehow not even attempt to build the entire `documentation` and `javadoc` trees instead?

Reply:
@joakime why you want to skip the module? With the property, Maven would process the module but skip the heavy work, so almost a no-op, but with module sanity checks (e.g. checking for circular deps, etc.).
Seems fast enough to me 😃

Reply:
@sbordet if you run the command @olamy referred to (`mvn -Pfast -ntp -V -B clean install -DskipTests -T6 -e`) then the build fails because the documentation module refuses to create an empty doc archive, so you get the maven failure of above.

Reply:
I tested the current `jetty-12.0.x` HEAD (commit 9ce414125de8d80d7c39cbda67fd600098f3bf6e) with a simulated deploy using the release profile.

``` shell
# Tip: start with a clean/empty $HOME/tmp/alt-repo/ directory
$ mvn clean deploy -Pci -Peclipse-release -Dmaven.build.cache.enabled=false -DskipTests -DaltDeploymentRepository=oss.sonatype.org::default::file:$HOME/tmp/alt-repo/
```

The deployed jetty-documentation artifacts are looking good ...

``` shell
[~/tmp/alt-repo]$ ls -la org/eclipse/jetty/documentation/jetty-documentation/12.0.2-SNAPSHOT/
total 17872
drwxrwxr-x 2 joakim joakim     4096 Sep 28 10:29 .
drwxrwxr-x 3 joakim joakim     4096 Sep 28 10:29 ..
-rw-rw-r-- 1 joakim joakim  4242911 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-html.zip
-rw-rw-r-- 1 joakim joakim      870 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-html.zip.asc
-rw-rw-r-- 1 joakim joakim       32 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-html.zip.md5
-rw-rw-r-- 1 joakim joakim       40 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-html.zip.sha1
-rw-rw-r-- 1 joakim joakim 13983283 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-javadoc.jar
-rw-rw-r-- 1 joakim joakim      870 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-javadoc.jar.asc
-rw-rw-r-- 1 joakim joakim       32 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-javadoc.jar.md5
-rw-rw-r-- 1 joakim joakim       40 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-javadoc.jar.sha1
-rw-rw-r-- 1 joakim joakim    14474 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1.pom
-rw-rw-r-- 1 joakim joakim      870 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1.pom.asc
-rw-rw-r-- 1 joakim joakim       32 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1.pom.md5
-rw-rw-r-- 1 joakim joakim       40 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1.pom.sha1
-rw-rw-r-- 1 joakim joakim     1669 Sep 28 10:29 maven-metadata.xml
-rw-rw-r-- 1 joakim joakim       32 Sep 28 10:29 maven-metadata.xml.md5
-rw-rw-r-- 1 joakim joakim       40 Sep 28 10:29 maven-metadata.xml.sha1
```

Reply:
> I tested the current `jetty-12.0.x` HEAD (commit [9ce4141](https://github.com/eclipse/jetty.project/commit/9ce414125de8d80d7c39cbda67fd600098f3bf6e)) with a simulated deploy using the release profile.
> 
> ```shell
> # Tip: start with a clean/empty $HOME/tmp/alt-repo/ directory
> $ mvn clean deploy -Pci -Peclipse-release -Dmaven.build.cache.enabled=false -DskipTests -DaltDeploymentRepository=oss.sonatype.org::default::file:$HOME/tmp/alt-repo/
> ```
> 
> The deployed jetty-documentation artifacts are looking good ...
> 
> ```shell
> [~/tmp/alt-repo]$ ls -la org/eclipse/jetty/documentation/jetty-documentation/12.0.2-SNAPSHOT/
> total 17872
> drwxrwxr-x 2 joakim joakim     4096 Sep 28 10:29 .
> drwxrwxr-x 3 joakim joakim     4096 Sep 28 10:29 ..
> -rw-rw-r-- 1 joakim joakim  4242911 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-html.zip
> -rw-rw-r-- 1 joakim joakim      870 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-html.zip.asc
> -rw-rw-r-- 1 joakim joakim       32 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-html.zip.md5
> -rw-rw-r-- 1 joakim joakim       40 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-html.zip.sha1
> -rw-rw-r-- 1 joakim joakim 13983283 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-javadoc.jar
> -rw-rw-r-- 1 joakim joakim      870 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-javadoc.jar.asc
> -rw-rw-r-- 1 joakim joakim       32 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-javadoc.jar.md5
> -rw-rw-r-- 1 joakim joakim       40 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1-javadoc.jar.sha1
> -rw-rw-r-- 1 joakim joakim    14474 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1.pom
> -rw-rw-r-- 1 joakim joakim      870 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1.pom.asc
> -rw-rw-r-- 1 joakim joakim       32 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1.pom.md5
> -rw-rw-r-- 1 joakim joakim       40 Sep 28 10:29 jetty-documentation-12.0.2-20230928.145928-1.pom.sha1
> -rw-rw-r-- 1 joakim joakim     1669 Sep 28 10:29 maven-metadata.xml
> -rw-rw-r-- 1 joakim joakim       32 Sep 28 10:29 maven-metadata.xml.md5
> -rw-rw-r-- 1 joakim joakim       40 Sep 28 10:29 maven-metadata.xml.sha1
> ```

No doubt it works in this case :).
The problem is the `fast` profile with the addition  of `<asciidoctor.skip>true</asciidoctor.skip>` it's now breaking.
Either we remove this or fix it but not leave it as broken.

Just try:
```
 mvn clean install -T6 -Pfast
```